src/style.scss

```diff
@@ -42,6 +42,12 @@ body {
   color: $color-text;
 }
 
+body {
+  margin: 0;
+  padding: 8px;
+  max-width: 100vw;
```

Comment:
This style is causing problems in the desktop mode. It causes the popup to open in a weird and annoying way.
Please remove it if it doesn't impact much the mobile version. Or change it in a way that does not affect the desktop version.

Thanks!

Reply:
Removed. Thank you for checking.

Reply:
Thanks for fixing. The issue was specifically with the `max-width: 100vw;` line. Feel free to keep tinkering and upload a new PR if you like. I'll merge this one and see if we can make our way into mobile :)